README.md

```diff
@@ -46,21 +55,15 @@ https://developers.google.com/google-apps/tasks/firstapp
 
 Scroll to the section named __Register your project__.
 
-And similar over here https://www.dropbox.com/developers/apps
 
 # Build the project
 
-For free version:
-
-    ./gradlew installFreeDebug
-
-Or with play services and location reminders:
-
-    ./gradlew installPlayDebug
-
-Same as above but including Dropbox at the moment:
-
-    ./gradlew installPlayBetaDebug
+```powershell
```

Comment:
never seen `powershell` as the type before. would have expected `bash` maybe.

As long as it works though :+1: